allennlp/data/fields/flag_field.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Dict, List
+
+from overrides import overrides
+
+from allennlp.data.fields.field import Field
+
+
+class FlagField(Field[Any]):
+    """
+    A class representing a flag, which must be constant across all instances in a batch.
+    This will be passed to a `forward` method as a single value of whatever type you pass in.
+
+    This is intended for use with boolean flags, but in principle it should also work with any other
+    type.
```

Comment:
This seems useful in more cases than just boolean flags.

Now, if you need a setting to be consistent between reader and model, you can put it into a `FlagField`. Before, you had to specify it twice in the config, giving opportunity to mess it up.

Reply:
Well, this specifically goes to `forward`, where things that you specify in a model config go to the model's constructor.

You're suggesting making the comment here less prescriptive?

Reply:
Yes

Reply:
I couldn't think of anything useful to say that wasn't as prescriptive, so I just removed the line entirely.